.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
This is the current conda-forge default. It is overridden by the jinja2 `set` statement at the top of the recipe

If we wanted the `.ci_support/` files to also display `3.10`, we would need to use a separate `conda_build_config.yaml` file instead of `set`. I chose `set` simply because that keeps all the changes in a single file, but I'm fine with either implementation